torch/distributed/_tensor/_dynamo_utils.py

```diff
@@ -1,6 +0,0 @@
-from torch._dynamo import allow_in_graph
-from torch.distributed._tensor.api import DTensor
-
-# dynamo/torch.compile utils for
-allow_in_graph(DTensor)
-allow_in_graph(DTensor.from_local)
```

Comment:
L5 is wrong, we should remove it. This triggers some failures during this refactor.
L6 has been covered by ```trace_rules.lookup```.
So we are good to remove this file.